calendar_event_report/wizard/weekly_report_wizard.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import datetime
+from odoo import models, fields, api
+
+
+class WeeklyReportWizard(models.TransientModel):
+    _name = 'weekly.report.wizard'
+
+    date_start = fields.Date(
+        string="Start Date",
+        required=True,
+        default=lambda self: self._get_current_monday()
+    )
+    date_end = fields.Date(
+        string="End Date",
+        required=True,
+        default=lambda self: self._get_current_friday()
+    )
+    room_id = fields.Many2one(
```

Comment:
resource_id and point to resource.resource of calendar_resource and not project_resource_calendat.room